src/RestControllers/FhirPatientRestController.php

```diff
@@ -78,7 +78,9 @@ public function getAll($search)
 
         $searchParam = array(
             'name' => $search['name'],
-            'dob' => $search['birthdate']
+            'dob' => $search['birthdate'],
+			'fname' => $search['fname'],
+			'lname' => $search['lname']
```

Comment:
Is there a standard terminology for what can be searched for via FHIR?

Reply:
Yes there is a set of terminologies: https://www.hl7.org/fhir/searchparameter-registry.html
I think fname and lname are not mentioned in this set. However the implementation for fname and lname was done in PatientService.php and thats why I added it in FhirPatientRestController.php.

Reply:
I have my code ready for search abiding by the given terminologies. I can push that code and revert the fname and lname if required.

Reply:
I suppose it depends on how strict we need to be. If we need to be strict, then could basically set fname and lname to also equal $search['name'] (so sending search for name would work as full name, last name, or first name). Will let FHIR superhero @sjpadgett weigh in in this one.

Reply:
Ok sure.

Reply:
> I have my code ready for search abiding by the given terminologies. I can push that code and revert the fname and lname if required.

Our rest services are setup for rest queries so yes revert.